chsdi/templates/htmlpopup/minergiegebaeude.mako

```diff
@@ -0,0 +1,27 @@
+<%inherit file="base.mako"/>
+
+<%def name="table_body(c, lang)">
+<%
+    if lang == 'de':
```

Comment:
What about RM?
You could use DE as fallback. Something like:
if FR
[*_fr]
elif EN
[*_en]
elif IT
[*_it]
else
[*_de]